schema_based_on_diagram.sql

```diff
@@ -51,4 +51,8 @@ CREATE TABLE medical_histories_treatments(
     FOREIGN KEY (medical_histories_id) REFERENCES medical_histories (id),
     FOREIGN KEY (treatments_id) REFERENCES treatments (id),
     PRIMARY KEY (medical_histories_id, treatments_id)
-);
\ No newline at end of file
+);
+
+--Creating index for the joined table to improve performance
+CREATE INDEX medical_histories_id_index ON medical_histories_treatments(medical_histories_id);
+CREATE INDEX treatments_id_index ON medical_histories_treatments(treatments_id);
```

Comment:
- [ ] Nice work, but you should create indexes on all foreign keys, not only on join tables as per the requirements.